static/scss/homepage.scss

```diff
@@ -783,46 +783,54 @@ blocks,
 .nav-tabs-content-wrapper .panel-main-content .panel-body {
 	padding-top:0;
 }
-
+/*---------------------------------------
+   SECTION:  FOOTER
+-----------------------------------------*/
 footer {
```

Comment:
i realize that you did not edit all of the styles under the `footer` element, but can you make the tabs 2 spaces for all of these definitions instead of 4? the rest of the file (and our other css files) use 2 spaces, not 4